tests/test_example_docs_scripts.py

```diff
@@ -236,38 +263,6 @@ def test_parametric_sim_twoproc():
     # Suppress the temporary directory
     shutil.rmtree( temporary_dir )
 
-def test_ionization_script_twoproc():
```

Comment:
This test has been moved a few lines above, and reuses the function `run_sim`, so as to reduce the amount of code.